core/storage/question/gae_models.py

```diff
@@ -260,6 +260,92 @@ def get_question_skill_links_by_skill_ids(
         )
         return question_skill_link_models, next_cursor_str
 
+    @classmethod
+    def get_question_skill_links_based_on_mastery_equidistributed_by_skill(
+            cls, total_question_count, skill_ids, degrees_of_mastery):
+        """Fetches the list of constant number of QuestionSkillLinkModels
+        linked to the skills, sorted by the difference between skill
```

Comment:
Is this the signed difference or the absolute value of the difference?

If the latter, say "absolute value of the difference". If the former, say "by the value of (skill difficulty - skill mastery)" or vice versa.

Ditto for Returns.

Reply:
Done.